product_docs/docs/epas/11/installing/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Installing EDB Postgres Advanced Server"
+navTitle: "Installing"
+redirects:
+ - /epas/11/epas_inst_linux/
+ - /epas/11/epas_inst_linux/installing_epas_using_edb_repository/
+navigation:
+- linux_x86_64
+- linux_ppc64le
+- windows
+- linux_install_details
+- windows_install_details
+---
+## [Linux x86-64 (amd64)](linux_x86_64)
+### Red Hat Enterprise Linux (RHEL) and derivatives
+- [RHEL 8](linux_x86_64/epas_rhel_8), [RHEL 7](linux_x86_64/epas_rhel_7)
```

Comment:
I think it should be listed to match the headings like this:
- RHEL 8 or Oracle Linux 8
- RHEL 7 or Oracle Linux

It should be matching the titles of install topics

Reply:
The landing page is closely tied to the Platform Compatibility page and is organized the same way. We need to add a stem sentence to make it a little clearer, but the expectation is that users will pick the OS they are using to go to the relevant topic. I think we want to continue to list the Oracle Linux OSs separately. I don't think we need for the landing page to match how we are handling the topics themselves.